static/js/subs.js

```diff
@@ -207,6 +207,12 @@ exports.rerender_subscriptions_settings = function (sub) {
     stream_data.update_subscribers_count(sub);
     stream_ui_updates.update_subscribers_count(sub);
     stream_ui_updates.update_subscribers_list(sub);
+
+    // Update navbar if needed
+    const filter = narrow_state.filter();
+    if (filter && filter._sub && filter._sub.stream_id === sub.stream_id) {
+        tab_bar.render_title_area();
+    }
```

Comment:
@YashRE42 this change is a duplicate of this block,
https://github.com/zulip/zulip/blob/821e25ffb01bb8153161a8f0474ee50f62151118/static/js/subs.js#L144-L148
I suggest to extract this block into a function (can be `tab_bar.update_title_area`).